src/backend/columnar/columnar_customscan.c

```diff
@@ -407,6 +458,28 @@ ColumnarIndexScanAddTotalCost(PlannerInfo *root, RelOptInfo *rel,
 }
 
 
+/*
+ * RecostColumnarSeqPath re-costs given seq path for columnar table with
+ * relationId.
+ */
+static void
+RecostColumnarSeqPath(RelOptInfo *rel, Oid relationId, Path *path)
+{
+	if (!enable_seqscan)
+	{
+		/* costs are already set to disable_cost, don't adjust them */
+		return;
+	}
+
+	path->startup_cost = 0;
+
+	/* seq scan doesn't support projection pushdown, so we will read all the columns */
```

Comment:
Note that it also doesn't support chunk group filtering, but that it's hard to cost that accurately. We can ignore that for now, but let's add a comment.